src/user/SCM_CVMix_tests.F90

```diff
@@ -264,7 +273,8 @@ subroutine SCM_CVMix_tests_buoyancy_forcing(sfc_state, fluxes, day, G, US, CS)
     ! Note CVMix test inputs give max sw rad in [Z C T-1 ~> m degC s-1]
     ! therefore must convert to [Q R Z T-1 ~> W m-2] by multiplying by Rho0*Cp
     ! Note diurnal cycle peaks at Noon.
-      fluxes%sw(i,J) = CS%Max_sw *  max(0.0, cos(2*PI*(time_type_to_real(DAY)/86400.0 - 0.5))) * CS%RHO0 * fluxes%C_p
+      fluxes%sw(i,J) = CS%Max_sw *  max(0.0, cos(2*PI*(time_type_to_real(DAY)/86400.0 - 0.5))) * &
```

Comment:
Not relevant to this PR but for some reason the case of `DAY` does not match the input `day`.  Also, I assume `time_type_to_real()` returns [s] (i.e. MKS) and not T?

Reply:
Yes, time_type_to_real does return values in [s], although perhaps we could revisit this.

Reply:
`DAY` case can be fixed down the road.